src/constants/entities/bow_wow.asm

```diff
@@ -0,0 +1,4 @@
+; constants for wIsBowWowFollowingLink
+BOW_WOW_IS_NOT_FOLLOWING_LINK  equ $00
+BOW_WOW_IS_FOLLOWING_LINK      equ $01
+BOW_WOW_HAS_BEEN_KIDNAPPED     equ $80
```

Comment:
Nitpick: maybe we could use somehow shorter names. After all, in the code we already know what we're manipulating from the wram variable name.

Something like:

```suggestion
BOW_WOW_NONE      equ $00
BOW_WOW_FOLLOWING equ $01
BOW_WOW_KIDNAPPED equ $80
```

But I'm also fine with leaving it as-this.

Reply:
This is absolutely better. I will adjust it to your suggestion.